src/utils/filterComment.js

```diff
@@ -0,0 +1,9 @@
+// 댓글과 대댓글을 재귀적으로 필터링하는 함수
+export const filterDeletedComments = (comments) => {
+  return comments
+    .filter((comment) => !comment.isDeleted)
```

Comment:
`comment`를 구조 분해하여 코드를 간결하게 작성할 수 있을 거 같아요.

```suggestion
    .filter(({ isDeleted }) => !isDeleted)
```